nifi-framework-api/src/test/resources/nar/MANIFEST-FULL.MF

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
```

Comment:
Do these manifest files support comments? If so, it would be possible to add the license header and avoid the need to exclude them from the license checking.

Reply:
As far as I know it does not support it, I tried using property file comment syntax which didn't work, and this seems to suggest it doesn't support it:
https://stackoverflow.com/questions/2025872/comment-lines-in-manifest-of-a-jar